smelly_python/command_line.py

```diff
@@ -14,13 +14,27 @@
 from .generator.md_generator import generate_md
 
 
+def get_grade(text):
+    """
+    Extracts the grade from the text exported by pylint.
+    :param: text the exported text
+    :return: the grade with one decimal place
+    """
+    search = re.search(r"Your code has been rated at (\d+)\.?(\d*)", text)
```

Comment:
If you use the: "Your code has been rated at (\d+\.?\d*)" you only have to use `search.group(1)` right instead of still splitting it etc. ?

Reply:
I changed this on purpose to remove the decimal if there is just .00 after, cause I found it ugly 😄

Reply:
If you're changing the code anyway can you change the double quotes to single quotes? :innocent:

Reply:
lgtm